packages/shared/src/hooks/useOrganization.tsx

```diff
@@ -72,10 +91,14 @@ type UseOrganizationReturn =
       domains: PaginatedResourcesWithDefault<OrganizationDomainResource>;
       membershipRequests: PaginatedResourcesWithDefault<OrganizationMembershipRequestResource>;
       memberships: PaginatedResourcesWithDefault<OrganizationMembershipResource>;
+      invitations: PaginatedResourcesWithDefault<OrganizationInvitationResource>;
     }
   | {
       isLoaded: boolean;
       organization: OrganizationResource | null;
+      /**
+       * @deprecated Use `invitations` instead
```

Comment:
@dimkl Introduced a deprecated utility function that logs a deprecation message for our customers. You should use it.

Reply:
on it